tests/test_main.py

```diff
@@ -1 +1,32 @@
-# write your code here
+from unittest import mock
+
+from app.main import cryptocurrency_action
+
+
+@mock.patch("app.main.get_exchange_rate_prediction")
+def test_prediction_is_higher_than_5(mocked_prediction):
+    mocked_prediction.return_value = 1.0
+
+    assert cryptocurrency_action(1.05) == "Do nothing"
```

Comment:
`1` = 0.95238... * 1.05, you already have checked when predicted value equals exactly 0.95 of initial value below but you still haven't check boundary values when prediction equals **exactly** 1.05 of initial value.

Reply:
fixed